koji-prometheus-exporter.py

```diff
@@ -286,6 +335,8 @@ def scrape():
             'koji_in_progress_tasks': koji_in_progress_tasks_family,
             'koji_waiting_tasks': koji_waiting_tasks_family,
             'koji_task_duration_seconds': koji_task_duration_seconds_family,
+            'koji_waiting_task_duration_seconds': koji_waiting_task_duration_seconds_family,
```

Comment:
This is a super nit-picky thing, but I think this should be `koji_task_waiting_duration_seconds` instead of `koji_waiting_task_duration_seconds`.

It is not the duration of "waiting tasks" it is the "waiting duration" of tasks.

All I'm going on here is https://prometheus.io/docs/practices/naming/

Reply:
ack, should I also change `koji_waiting_tasks` to `koji_tasks_waiting` or even `koji_tasks_in_waiting` ?

Reply:
Made the suggested change and also applied it to `koji_in_progress_task_duration_seconds`. I did not change `koji_waiting_tasks` nor `koji_in_progress_tasks` as those were existing entities.